BUILD

```diff
@@ -166,30 +166,40 @@ gazelle_binary(
     languages = DEFAULT_LANGUAGES + ["@bazel_gazelle//language/bazel/visibility"],
 )
 
-# Ignore the node_modules dir
+## Ignore the node_modules dir
```

Comment:
This may not be necessary: https://github.com/bazel-contrib/bazel-gazelle/commit/1dbcd58297322ddeeafbfd006b288cede1892352

Reply:
oh neat!